src/ipc/CMakeLists.txt

```diff
@@ -10,5 +10,9 @@ endif()
 add_local_sources(sof
 	ipc.c
 	handler.c
-	dma-copy.c
 )
+
+if (CONFIG_TRACE)
+add_local_sources(sof
```

Comment:
Please use indention in CMake just like in C (increase indent in ifs, functions etc)

Reply:
Done.